src/main/java/duke/Task.java

```diff
@@ -0,0 +1,46 @@
+package duke;
+
+import java.time.LocalDate;
+
+public class Task {
+    protected String itself;
+    protected boolean isDone;
+
+    public Task(String itself) {
+        this.itself = itself;
+        this.isDone = false;
+    }
+
+    public String getStatus() {
+        return isDone ? "Y" : "N";
+    }
+
+    public void donelah() {
```

Comment:
Perhaps you can use camelCase for this method name? Generally, the naming convention for setter methods are setX().